idea/testData/quickfix/expressions/removeUselessIsCheckInWhen.kt

```diff
@@ -0,0 +1,6 @@
+// "Remove useless is check" "false"
+fun foo(a: String) {
+    when (1) {
+        <caret>is Int -> { }
```

Comment:
Well, it's an exceptional case but here we can also do something :). If a condition in when branch is definitely true, we can replace the condition with `else` and delete all subsequent branches because they are unreachable anyway. If the condition is definitely false (add test for this case), we can delete this when branch.